ereuse_devicehub/resources/device/models.py

```diff
@@ -237,10 +257,12 @@ def physical_possessor(self):
         and :class:`ereuse_devicehub.resources.action.models.Receive`
         changes it.
         """
-        from ereuse_devicehub.resources.action.models import Receive
-        with suppress(LookupError):
-            action = self.last_action_of(Receive)
-            return action.agent
+        pass
+        # TODO @cayop uncomment this lines for link the possessor with the device
+        # from ereuse_devicehub.resources.action.models import Receive
+        # with suppress(LookupError):
```

Comment:
LookupError example when you use last_action_of

Reply:
Good better solution, thanks